apps/src/applab/setPropertyDropdown.js

```diff
@@ -167,16 +172,22 @@ PROP_NAMES[ElementType.SLIDER] = [
 ];
 
 // Create a mapping of PROPS_PER_TYPE[elementType][friendlyName] => prop info
+// and a mapping of PROP_NAMES_PER_TYPE[elementType] => array of properties
 var PROPS_PER_TYPE = {};
+var PROP_NAMES_PER_TYPE = {};
 Object.keys(PROP_NAMES).map(function (elementType) {
```

Comment:
Or, this might be expressed better with [lodash's `_.mapValues`](https://lodash.com/docs#mapValues):

```javascript
let PROPS_PER_TYPE = _.mapValues(PROP_NAMES, (elementNames, elementType) => {
  // compute and return props for elementType
});
```